.github/workflows/ISSUE_TEMPLATE/bug_report.md

```diff
@@ -0,0 +1,47 @@
+---
+name: Bug report
+about: Create a report to help us improve
+title: ''
+labels: bug
+assignees: ''
+
+---
+
+## Bug description
+
+<!-- A clear and concise description of what the bug is. -->
+
+- Would you like to work on a fix? [y/n]
+
+## To Reproduce
+
+Steps to reproduce the behavior:
+
+1. ...
+2. ...
+3. ...
+4. ...
+
+<!-- Make sure you are able to reproduce the bug in the main branch, too. -->
+<!-- Make sure the issue is not already reported. -->
```

Comment:
Maybe we can make this a tick box option, like "I have searched the existing issues (closed & open) to make sure the issue hasn't already been reported?"

Reply:
Good idea! I've updated the comments to checkboxes